### PR TITLE
Align with latest PSA Attestation API Overview

### PR DESCRIPTION
**Pull request by thomas-fossati**
This change also includes a couple of minor edits:
- Update EAT ref to WG draft;
- Add BCP14 boilerplate.

### CODE REVIEW

File: psa-tokens/draft-tschofenig-rats-psa-token.md [hunk 1]
```diff
@@ -173,27 +171,28 @@ NSPE
...
+| Boot Seed | Yes | Represents a random value created at system boot time that will allow differentiation of reports from different boot sessions. |
+| Software Components | Yes (unless the No Software Measurements claim is specified) | A list of software components that represent all the software loaded by the PSA Root of Trust. This claim is needed for the rules outlined in {{PSA-SM}}. The software components are further explained below. |
+| No Software Measurements | Yes (if no software components specified) | In the event that the implementation does not contain any software measurements then the Software Components claim above can be omitted but instead it will be mandatory to include this claim to indicate this is a deliberate state. This claim is intended for devices that are not compliant with {{PSA-SM}}.|
 {: #info-model title="Information Model of PSA Attestation Claims."} 
 
 The PSA lifecycle states consist of the following values:
 
-- UNKNOWN (0x1000u)
+- UNKNOWN (0x0000u)
```
Comment (adrianlshaw) [inline psa-tokens/draft-tschofenig-rats-psa-token.md]: PSA_LIFECYCLE_UNKNOWN (0x0000u)

File: psa-tokens/draft-tschofenig-rats-psa-token.md [hunk 2]
```diff
@@ -173,27 +171,28 @@ NSPE
...
+- ASSEMBLY_AND_TEST (0x1000u)
```
Comment (adrianlshaw) [inline psa-tokens/draft-tschofenig-rats-psa-token.md]: PSA_LIFECYCLE_ASSEMBLY_AND_TEST (0x1000u)

File: psa-tokens/draft-tschofenig-rats-psa-token.md [hunk 3]
```diff
@@ -173,27 +171,28 @@ NSPE
...
 - PSA_ROT_PROVISIONING (0x2000u)
```
Comment (adrianlshaw) [inline psa-tokens/draft-tschofenig-rats-psa-token.md]: PSA_LIFECYCLE_PSA_ROT_PROVISIONING (0x2000u)

File: psa-tokens/draft-tschofenig-rats-psa-token.md [hunk 4]
```diff
@@ -173,27 +171,28 @@ NSPE
...
 - SECURED (0x3000u)
```
Comment (adrianlshaw) [inline psa-tokens/draft-tschofenig-rats-psa-token.md]: PSA_LIFECYCLE_SECURED (0x3000u)

File: psa-tokens/draft-tschofenig-rats-psa-token.md [hunk 5]
```diff
@@ -173,27 +171,28 @@ NSPE
...
 - NON_PSA_ROT_DEBUG (0x4000u)
```
Comment (adrianlshaw) [inline psa-tokens/draft-tschofenig-rats-psa-token.md]: PSA_LIFECYCLE_NON_PSA_ROT_DEBUG (0x4000u)

File: psa-tokens/draft-tschofenig-rats-psa-token.md [hunk 6]
```diff
@@ -173,27 +171,28 @@ NSPE
...
 - RECOVERABLE_PSA_ROT_DEBUG (0x5000u)
```
Comment (adrianlshaw) [inline psa-tokens/draft-tschofenig-rats-psa-token.md]: PSA_LIFECYCLE_RECOVERABLE_PSA_ROT_DEBUG (0x5000u)

File: psa-tokens/draft-tschofenig-rats-psa-token.md [hunk 7]
```diff
@@ -173,27 +171,28 @@ NSPE
...
-- PSA_LIFECYCLE_DECOMMISSIONED (0x6000u)
+- DECOMMISSIONED (0x6000u)
```
Comment (adrianlshaw) [inline psa-tokens/draft-tschofenig-rats-psa-token.md]: PSA_LIFECYCLE_DECOMMISSIONED (0x6000u)

**Comment by thomas-fossati** [comment]
> I suggest adding a sentence to the introduction along the following lines: "Further details on concepts expressed below can be found within the [PSA-SM]."

Done in d8348d4f83d7a426a2350e61e23bf9207dfdf300

**Comment by thomas-fossati** [comment]
> One additional alignment: in the Software Component table, SignerID is not mandatory, but it should be present to comply with PSA-SM.

Good catch!  Done in d8348d4f83d7a426a2350e61e23bf9207dfdf300

**Comment by adrianlshaw** [comment]
We deleted a paragraph about which software components field is mandatory or optional, but actually I think that deletion is detrimental. I will create a PR to address this.

**Comment by adrianlshaw** [comment]
Actually I changed my mind about this. Made some smaller tweaks instead, see #43.

**Pull request merged**